.ci/.jenkins-schema.groovy

```diff
@@ -0,0 +1,12 @@
+---
+agents:
+  - REPO: "apm-agent-dotnet"
+  - REPO: "apm-agent-go"
+    SPEC_FILEPATH: "internal/apmschema/jsonschema"
+  - REPO: "apm-agent-java"
+    SPEC_FILEPATH: "apm-agent-core/src/test/resources/apm-server-schema/current"
+  - REPO: "apm-agent-nodejs"
+  - REPO: "apm-agent-php"
+  - REPO: "apm-agent-python"
+  - REPO: "apm-agent-ruby"
```

Comment:
@estolfo  pointed me to use https://github.com/elastic/apm-agent-ruby/tree/master/spec/fixtures

```suggestion
  - REPO: "apm-agent-ruby"
    SPEC_FILEPATH: "spec/fixtures"
```